specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/search.json

```diff
@@ -819,29 +773,12 @@
           }
         }
       },
-      "required": [
```

Comment:
I think I understand the reason to remove them as required - I could be wrong, but are you trying to allow "SearchService" to be an acceptable request payload for the "patch" operation? In that case, it might help to create some type like `SearchServicePatchParameters`?

Reply:
Right, I want the client programming model to be simpler by using the same type for both operations.

Reply:
But that way "location" and "sku" are not validated in the create scenarios. @veronicagg Do you have any guidelines which is the recommended way?

Reply:
They are still validated by the RP. This is just a tradeoff between stronger type safety on the client versus fewer classes for the customer to deal with.

Reply:
Discussed with Veronica offline - @brjohnstmsft Do you think you can add some descriptions to "sku" property so that users will know this is required in a create scenario?

Reply:
Sure, I'll make that change.